ocs_ci/ocs/scale_noobaa_lib.py

```diff
@@ -68,16 +72,15 @@ def check_all_obc_reached_bound_state_in_kube_job(
         # Get kube_job obj and fetch either all OBC's are in Bound state
         # If not bound adding those OBCs to obc_not_bound_list
         job_get_output = kube_job_obj.get(namespace=namespace)
-        if job_get_output is not None and len(job_get_output) == no_of_obc:
-            for i in range(no_of_obc):
-                status = job_get_output["items"][i]["status"]["phase"]
-                log.info(
-                    f"obc {job_get_output['items'][i]['metadata']['name']} status {status}"
+        for i in range(no_of_obc):
```

Comment:
Do we know what is the approximate time('y' seconds) to bind 'x' number of OBCs? 
IMO, having to start the job and immediately check for PVCs/OBCs being bound is adding unnecessary ops. Why not first wait for a minimum 'y' seconds and then start validating. Also, what value addition do we get by logging 'in progress' OBCs being bound. 
I guess this is similar to `check_all_pvc_reached_bound_state_in_kube_job`, so tagging @ramkiperiy

Reply:
@keesturam PVC/OBC creation uses kube_job which is bulk create and in-case of 760+ PVC creations even waiting for particular time might not be sufficient, In this function there is a while True loop which has 2 lists(bound_list and not_bound_list), in each while loop not_bound_list will be updated with PVCs not in Bound state after that there is a waittime in each iteration. PVC Bound validation is done using kube_job get cmd output yaml file, this is faster.
1. In-case of no data in not_bound_list(i.e. all PVCs are Bound) there is a break for while True loop
2. In-case of all iteration count is reached with waittime in each iteration break is issues for while True loop
3. In failure situation list of PVCs which are not in Bound state will be printed in logs.

If we wait for particular 'y' seconds and starting validation might be problem since we need more iteration or waittime.
Does this approach looks good ? If there is some logic which can be more effective we can try it.

Reply:
@keesturam The wait time is vary base on number of obc creation.  Waiting for a fix time before validating obc's status will not be sufficient.  While loop is a better handling for timing in case of obc creation.

Reply:
Thank you @ramkiperiy @tiffanyn108 for the explanation. Let us leave this flow as is.